gatsby_fp/src/components/fieldComponents/education.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+export default function Education ({educationList}){
+
+  return(
+    <>
+    <div className="sectiontitle">Education</div>
+        <div>
+        <ul>
+          {educationList.map((element, index) => (
+            <li key={index} dangerouslySetInnerHTML={{__html: `${element.deg} (${element.dty_comp})
```

Comment:
Assigning the index to keys is an antipattern. It may have a performance impact in some cases, and will cause dynamic components to behave unexpectedly when indices change.

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318